package-overrides/github/angular/bower-material@0.9.0-rc1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "main": "index",
```

Comment:
@guybedford - so what does this shim addition mean exactly? I realize that Angular Material application depend upon the `angular-material.css` and the CSS plugin. But this shim does not load the CSS... so what is the shim doing?

```json
"shim": {
    "angular-material": {
      "deps": [ "./angular-material.css!" ]
    }
  }
```